server/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -121,14 +121,6 @@ public SearchRequestBuilder setWaitForCheckpoints(Map<String, long[]> waitForChe
         return this;
     }
 
-    /**
-     * Set the timeout for the {@link #setWaitForCheckpoints(Map)} request.
-     */
-    public SearchRequestBuilder setWaitForCheckpointsTimeout(final TimeValue waitForCheckpointsTimeout) {
```

Comment:
@original-brownbear this is used in stateless (in a test at least). Can you revert?